templates/docs/patterns/lists.md

```diff
@@ -20,18 +20,18 @@ items than the basic lists.
 View example of the list pattern
 </a></div>
 
-### Ticked
+### Status
 
-Add the class `.is-ticked` to each list item to display tick icons.
+Add the `.is-ticked` or `.is-crossed` classes to each list item to display tick/cross icons.
 
 <div class="p-notification--information">
   <p class="p-notification__response">
-    <span class="p-notification__status">Note:</span>The color of the tick icon is set by the <code>$color-accent</code> variable in <code>settings.scss</code>.
+    <span class="p-notification__status">Note:</span>The color of the tick icon is set by the <code>$color-accent</code> variable in <code>settings.scss</code>, and the color of the cross icon is set by the <code>$color-accent-background</code> variable.
```

Comment:
Is this true? I guess icons are green/red as default positive/negative colors?

Reply:
Argh - this is a leftover from earlier in the day...